### DISCUSSION TITLE
transformations not available on imported svg

### DISCUSSION THREAD

**Discussion by andrearicci** [friction2d/friction#248] [Q&A] [answered]
Hi,
I just started using friction. I need to do a morph on an svg I made in inkscape. So I started to import the svg, and that's fine, but it's very small on th ecanvas, and I can't find a way to scale it: the transform menù is just grayed out. So I'm stuck at my first attempt.
Thank you for any help.

**Comment by rodlie** [comment, marked as answer]
`s` or use the transforms in properties or timeline.

**Comment by irwant** [comment]
>  but it's very small on th ecanvas 

![Screenshot_2024-09-17_19-14-21](https://github.com/user-attachments/assets/2bac9923-bcf6-4bb5-bf57-7f7fb20b9ddb)
make sure before import the SVG file into friction, you must set up the canvas size on your Inkscape. in this sample, I'm using resolution FHD, and then change the value for scale to 1. I recommended import file SVG on the menu than drag n drop.

**Reply by andrearicci** [reply]
thank you``

**Comment by andrearicci** [comment]
Thank you all.
S works. 
Still Y wonder why the edit→transform menù is grayed out whit ojects selected. Do I need to selct an object AND a timeline? (not in front of th epc to test it myself at the moment)

**Reply by rodlie** [reply]
Sorry about that, the transform menu has no function (bad UX), it has never been removed. Enabling them IMHO has no purpose as they are interactive actions you trigger through a shortcut while in the canvas.

Open for suggestions, and yes proper interactive transforms will be added in a later version (~v1.2+).

I will start working on basic documentation next week.

**Reply by andrearicci** [reply]
thank you

**Comment by andrearicci** [comment]
thank you all
